evaluator/evaluate_packages.go

```diff
@@ -49,12 +49,12 @@ func lazySavePackages(tx *gorm.DB, names, evras []string) error {
 	}
 	defer utils.ObserveSecondsSince(time.Now(), evaluationPartDuration.WithLabelValues("lazy-package-save"))
 
-	name2ID, err := loadPkgName2IDMap(tx, names)
+	name2ID, nameID2Hashes, err := loadPackagesInfos(tx, names)
```

Comment:
Hmm, is name2id 1:1 or N:1 relation? If 1:1 can we save some memory by putting both into the same map?